src/common/host_device_vector.h

```diff
@@ -62,8 +62,9 @@ template <typename T> struct HostDeviceVectorImpl;
 template <typename T>
 class HostDeviceVector {
  public:
-  explicit HostDeviceVector(size_t size = 0, int device = -1);
+  explicit HostDeviceVector(size_t size = 0, int device = -1, T v = T());
```

Comment:
I think I still prefer it in the order (size, v, device) for consistency with std vector.

Reply:
Done.